### ISSUE TITLE
Piping to gcloud compute ssh command does not work on new Debian instances

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prototipando/google-cloud-sdk#65]
```
This recently stopped working on new Debian instances. It seems to have 
something to do with which image the instance is based on, but I cannot be 100% 
sure. I tried this a few minutes ago as of this writing:

Instance based on backports-debian-7-wheezy-v20140718:

$ echo Hello | gcloud compute ssh --zone=us-central1-b INSTANCE_NAME --command 
cat
Warning: Permanently added '107.178.####' (RSA) to the list of known hosts.
Warning: Permanently added '107.178.####' (RSA) to the list of known hosts.
Hello

Instance based on backports-debian-7-wheezy-v20140814 - brand new instance, no 
configuration on my part, created using the defaults in the developer console, 
running the f1-micro machine type:

$ echo Hello | gcloud compute ssh --zone=us-central1-b instance-1 --command cat
Warning: Permanently added '23.251.####' (RSA) to the list of known hosts.
Warning: Permanently added '23.251.####' (RSA) to the list of known hosts.

Note the absense of "Hello" in the latter. Getting a shell on the latter using 
the ssh command works as expected, so it's not a firewall problem.

```

Original issue reported on code.google.com by `haakone...@gmail.com` on 26 Aug 2014 at 9:53

**Comment by GoogleCodeExporter** [comment]
Original comment by `rdayal@google.com` on 27 Aug 2014 at 1:36

**Comment by GoogleCodeExporter** [comment]
Original comment by `jasm...@google.com` on 28 Aug 2014 at 2:41
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This is fixed in 0.9.69 (possibly earlier).
```

Original comment by `z...@google.com` on 14 Jul 2015 at 9:04
- Changed state: **Fixed**
